core/iwasm/interpreter/wasm_interp_classic.c

```diff
@@ -2315,7 +2321,7 @@ wasm_interp_call_func_bytecode(WASMModuleInstance *module,
 
                 /**
                  * type check. compiler will make sure all like
-                 * (call_indirect (type $x) (i32.const 1))
+                 * (call_indirect (type $x) (it.const 1))
```

Comment:
`i32.const` ?

Reply:
`it` is short for index type, depends on the table flag, it can be i32 or i64